Hearty-Vegetable-lasagana.md

```diff
@@ -1,15 +1,27 @@
 #**Hearty Vegetable Lasagna**
 
-![](http://www.feastsfromthepantry.com/wordpress/wp-content/uploads/2011/08/tumblr_lq8zjnW2T71qgmbh4.jpg)
+![](http://images.media-allrecipes.com/userphotos/560x315/620560.jpg  "Lasagana")
 
-#### __*Prep Time:*__ 25 minutes    
-### __*Ready in:*__ 1 Hour 40 minutes
-### __*Cook Time:*__ 1 Hour        
-### __*Serving:*__ 12
+Ready in|Prep Time |Cook time|Serving
+---|---|---|---
+1 Hour 40 minutes|25 minutes|1 Hour|12     
 
-"A pound of fresh mushrooms along wit lost of bell peppers and onions gives full-bodied flavor to this veggie-packed lasagna. Layer with noodles and a creamy blend of ricotta and mozzarellla cheeses."
+-"A pound of fresh mushrooms along wit lost of bell peppers and onions gives full-bodied flavor to this veggie-packed lasagna. Layer with noodles and a creamy blend of ricotta and mozzarellla cheeses."
+
+##**Ingredients**
+| | |
+|---|---|
+1 (16 ounce) package lasagna noodles | 2 (26 ounce) jars pasta sauce
+1 pound fresh mushrooms, sliced | 1 teaspoon dried basil
+3/4 chopped green pepper | 1 (15 ounce) container part-skim ricotta cheese
+3/4 cup chopped onion | 4 cups shredded mozzarella cheese
+3 cloves garlic, minced | 2 eggs
+2 tablespoons vegetable oil | 1/2 cup greated Parmesan cheese 
+
+##**Directions**
```

Comment:
In general you don't want to style the titles, the idea is that '#' will indicate top level (`<h1>`), `##` will be an `<h2>` and so on so we shouldn't add `**` to titles